frontend/javascripts/oxalis/model/sagas/volumetracing_saga.tsx

```diff
@@ -130,6 +135,25 @@ function* warnOfTooLowOpacity(): Saga<void> {
   }
 }
 
+function* warnTooLargeSegmentId(): Saga<void> {
+  yield* take("INITIALIZE_VOLUMETRACING");
+  let oldSegmentId = yield* select((state) => enforceActiveVolumeTracing(state).activeCellId);
+  while (true) {
+    yield* take(["SET_ACTIVE_CELL", "CREATE_CELL"]);
+    const newSegmentId = yield* select((state) => enforceActiveVolumeTracing(state).activeCellId);
+    if (oldSegmentId === newSegmentId) {
```

Comment:
I think, this logic is flawed. 1) oldSegmentId is never updated. 2) If I select an already active segment id, the condition will eval to true, I think? So, if the warning can also appear for uncritical segment ids (e.g., 3 is selected and I select it again somehow).

What do you think about using the "requested segment id" in the actions? I think, both set-active-cell and create-cell get a segment id parameter (probably named differently). That id could be checked against `maxSegmentId` then.

Reply:
> I think, this logic is flawed. 1) oldSegmentId is never updated. 2) If I select an already active segment id, the condition will eval to true, I think? So, if the warning can also appear for uncritical segment ids (e.g., 3 is selected and I select it again somehow).

Oh thanks for catching this flaw. :pray: 

> What do you think about using the "requested segment id" in the actions? I think, both set-active-cell and create-cell get a segment id parameter (probably named differently). That id could be checked against maxSegmentId then.

:+1: